loader/loader.go

```diff
@@ -467,6 +467,8 @@ func convertField(
 	switch fieldTag {
 	case "":
 		return data, nil
+	case "healthcheck":
+		return loadHealthcheck(data)
```

Comment:
I think we're planning on phasing out the `compose:` tags, and using the mapstructure way of converting them (by creating a type), but I think it's fine for now.